### PR TITLE
Update featured page

### PR DESCRIPTION
**Pull request by claration**
- Added more recent tweaks
- Updated existing tweaks with their proper repo

### CODE REVIEW

File: featured-ipad-src.json [hunk 1]
```diff
@@ -13,22 +13,22 @@
                "repoName":"Chariz"
             },
             {
-               "url":"https://featuredpage.getsileo.app/banners/com.tr1fecta.akara.png",
-               "title":"Akara",
-               "package":"com.tr1fecta.akara",
-               "repoName":"Chariz"
+               "url":"https://featuredpage.getsileo.app/banners/com.smokin1337.satellite.jng",
```
Comment (m1stadev) [inline featured-ipad-src.json]: Typo, but nevertheless please have the banners in JPG format.

File: featured-iphone-src.json [hunk 2]
```diff
@@ -48,43 +48,43 @@
...
+                      "packageIcon":"https://featuredpage.getsileo.app/icons/com.cameronkatri.quickactions.png"
                    },
                    {
                       "class":"FeaturedPackageView",
-                      "package":"xyz.royalapps.prokeys",
-                      "packageName":"ProKeys",
-                      "packageAuthor":"Justin Proulx",
+                      "package":"com.initwithframe.velvet",
+                      "packageName":"Velvet",
+                       "packageAuthor":"initWithFrame",
```
Comment (m1stadev) [inline featured-iphone-src.json]: Extra space here?

File: featured-ipad-src.json [hunk 3]
```diff
@@ -13,22 +13,22 @@
...
+               "title":"Satellite",
+               "package":"com.smokin1337.satellite",
+               "repoName":"Havoc"
             },
             {
                "url":"https://featuredpage.getsileo.app/banners/com.jontelang.snapper2.packix.jpg",
                "title":"Snapper 2",
```
Comment (m1stadev) [inline featured-ipad-src.json:23]: Please change this package to something else.

File: featured-ipad-src.json [hunk 4]
```diff
@@ -13,22 +13,22 @@
...
                "package":"com.jontelang.snapper2.packix",
-               "repoName":"Packix"
+               "repoName":"Havoc"
             },
             {
                "url":"https://featuredpage.getsileo.app/banners/com.bossgfx.elysian.jpg",
                "title":"Elysian",
```
Comment (m1stadev) [inline featured-ipad-src.json:29]: Please change this package to something else.

File: featured-ipad-src.json [hunk 5]
```diff
@@ -97,43 +97,43 @@
...
+                     "package":"lunotech11.legizmo.jupiter",
+                     "packageName":"Legizmo Jupiter",
+                     "packageAuthor":"lunotech11",
+                     "repoName":"Chariz",
+                     "packageIcon":"https://featuredpage.getsileo.app/icons/lunotech11.legizmo.jupiter.png"
                   },
                   {
                      "class":"FeaturedPackageView",
                      "package":"com.ryant453.groups",
                      "packageName":"Groups",
```
Comment (m1stadev) [inline featured-ipad-src.json:125]: Please change this package.

**Review by m1stadev — CHANGES_REQUESTED** [review]

**Review by elihwyma — APPROVED** [review]
lgtm

**Pull request merged**